### PR TITLE
fix  golint in staging/src/k8s.io/apiserver/pkg/admission/plugin/ include namespace/lifecycle,webhook/mutating,webhook/validating

### PR DESCRIPTION
**Pull request by tanshanshan**
**What this PR does / why we need it**:
fix golint in
staging/src/k8s.io/apiserver/pkg/admission/plugin/namespace/lifecycle
staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating	
staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
refer #68026

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/admission/plugin/initialization/initialization.go [hunk 1]
```diff
@@ -56,13 +56,14 @@ type initializerOptions struct {
...
-type InitializationConfig interface {
+// Config specifies initialization config
+type Config interface {
 	Run(stopCh <-chan struct{})
 	Initializers() (*v1alpha1.InitializerConfiguration, error)
 }
 
 type initializer struct {
-	config     InitializationConfig
+	config     Config
```
[the remark this answers is not in the document]
Comment (tanshanshan) [reply staging/src/k8s.io/apiserver/pkg/admission/plugin/initialization/initialization.go]: yeah,
staging/src/k8s.io/apiserver/pkg/admission/plugin/initialization/initialization.go:59:6: type name will be used as initialization.InitializationConfig by other packages, and that stutters; consider calling this Config

[the remark this answers is not in the document]
Comment (stewart-yu) [reply staging/src/k8s.io/apiserver/pkg/admission/plugin/initialization/initialization.go]: IMO, This change are wrong

[the remark this answers is not in the document]
Comment (stewart-yu) [reply staging/src/k8s.io/apiserver/pkg/admission/plugin/initialization/initialization.go]: keep `InitializationConfig `, and add comments

**Pull request merged**